Projeto Físico/consultas.sql

```diff
@@ -0,0 +1,6 @@
+-- quantidade de clientes que compraram passagem para cada destino num determinado ano
+SELECT LOCAL_DESTINO, extract(year from DATA_DESTINO), COUNT(*) FROM PASSAGEM
+-- em todos os anos
+GROUP BY LOCAL_DESTINO, extract(year from DATA_DESTINO)
+-- em determinado ano
+HAVING extract(year from DATA_DESTINO)=1856;
```

Comment:
Essa filtragem deveria ser feita antes do agrupamento, usando WHERE

```suggestion
WHERE extract(year from DATA_DESTINO)=1856;
```
Pra manter o HAVING, a filtragem deveria ser sobre o agrupamento. Por exemplo, [...] com quantidade de passagens para o destino maior que X

Reply:
Concordo com Malu, além disso, na consulta tu não faz exatamente a descrição, porque tu projeta o ano além da quantidade de passagens. 

Poderia mudar para: projetar a quantidade de passagens para um local de destino num determinado ano quando houvesse mais do que duas passagens, por exemplo.